### PR TITLE
Patch: upgrade-omero56

### PR DESCRIPTION
**Pull request by luiskuhn**
Patch for bioimage DM infrastructure upgrade. Improved documentation and included support for both image files and folders in metadata TSV file

### CODE REVIEW

File: drop-boxes/register-omero-metadata/register-omero.py [hunk 1]
```diff
@@ -61,7 +63,10 @@
 INCOMING_DATE_FORMAT = '%d.%m.%Y'
 OPENBIS_DATE_FORMAT = '%Y-%m-%d'
 
-PROPPERTY_FILTER_LIST = ["IMAGE_FOLDER_PATH", "INSTRUMENT_USER", "IMAGING_DATE", "SAMPLE_ID", "OMERO_TAGS", "ETL_TAG"]
+# For fast validation without parser object
+REQUIRED_PROPPERTY_LIST = ["IMAGE_DATA_PATH", "IMAGING_MODALITY", "IMAGED_TISSUE", "INSTRUMENT_MANUFACTURER", "INSTRUMENT_USER", "IMAGING_DATE"]
+# To filter property list before pushing key-value pair to OMERO server
+PROPPERTY_FILTER_LIST = ["IMAGE_DATA_PATH", "INSTRUMENT_USER", "IMAGING_DATE", "SAMPLE_ID", "OMERO_TAGS", "ETL_TAG"]
```
Comment (wow-such-code) [inline drop-boxes/register-omero-metadata/register-omero.py]: It is not 100% clear to me from this description what is filtered out. E.g. is Image_Data_Path necessary but it doesn't end up in omero?

Comment (wow-such-code) [reply drop-boxes/register-omero-metadata/register-omero.py]: also: PROPPERTY_FILTER_LIST is not used, I think?

Comment (wow-such-code) [reply drop-boxes/register-omero-metadata/register-omero.py]: NIT: property with a single P

Comment (luiskuhn) [reply drop-boxes/register-omero-metadata/register-omero.py]: correct, it is to filter out the properties that we don't want to put as key-value pairs in OMERO

**Review by wow-such-code — DISMISSED** [review]
looks good to me!

**Review by wow-such-code — APPROVED** [review]
looks even better now ;)

**Pull request merged**